Backpack-SwiftUI/SponsoredBanner/Classes/BPKSponsoredBanner.swift

```diff
@@ -0,0 +1,171 @@
+//
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright © 2023 Skyscanner Ltd. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.a=pache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+public struct BPKSponsoredBanner: View {
+    let logo: Image?
+    let title: String?
+    let subheadline: String?
+    let callToAction: CallToAction?
+    let bodyText: String?
+    let variant: Variant
+    let backgroundColor: Color
+    
+    @State var showTerms: Bool
```

Comment:
Personally I find this name feeling a bit incomplete, I'd be okay with a longer more descriptive name `showTermsAndConditions` but happy to hear what others think on short vs long names here 😄